src/styles/index.css

```diff
@@ -20,9 +20,17 @@
   :root {
     --p-spacing-focus-ring: 2px;
     --p-spacing-focus-ring-offset: 1px;
-
     --p-radius-default: 6px;
     --p-scrollbar-width: 12px;
+    --destructive: 4 74% 49%;
+    --positive: 142 71% 45%;
+    --neutral: 240 5% 46%;
+    --warning: 40 92% 50%;
```

Comment:
Generally i'd suggest that global tokens have some sort of prefix/namespace. That way we don't run the risk (or we have less risk) or them being overridden by another library or a 3rd party extension and also adds css to the root.

Reply:
We only need these for the tailwind plugin yeah? Otherwise we could just put these values in the tokens below?